src/components/repository-profile.component.js

```diff
@@ -79,10 +79,33 @@ const styles = StyleSheet.create({
   icon: {
     paddingBottom: 20,
   },
+  languageInfo: {
+    flexDirection: 'row',
+    marginTop: 35,
+  },
+  languageInfoTitle: {
+    color: colors.white,
+    paddingLeft: 3,
+    fontSize: normalize(10),
+    ...fonts.fontPrimary,
+  },
 });
 
 export const RepositoryProfile = ({ repository, starred, navigation }: Props) =>
   <View style={styles.container}>
+    {repository.language !== null &&
```

Comment:
Yep they return null (I had also assumed they would be empty string but that doesn't seem to be the case). Here's an example of an empty repo (no language) 
https://api.github.com/repos/housseindjirdeh/test-repo